examples/traits/pdb/README.md

```diff
@@ -0,0 +1,21 @@
+# Camel-k PDB Trait
+
+This is an example of a Pod Distruption Budget for an integration pod running on the cluster. 
+You can set a  `minumum available` or a `maximum unavailable` pods, it all depends on your eviction policy.
+
+ **Note:** You can only use one trait command. Either a `minimum available` or a `maximum unavailable`
```

Comment:
I think we should comment that this trait is better tested on a Cluster with more than one node available, as it would be difficult to show it in action on a single node cluster local machine.